### ISSUE TITLE
Error occurred when pass argument to ScreenRoute method

### ISSUE THREAD

**Issue by tnitsiri** [nubank/nuvigator#53]
When I run flutter pub run build_runner watch --delete-conflicting-outputs
generate file not created and show some error outputs

line 1, column 924: Expected to find '}'.
  ╷
1 │ Future<Object> toMerchantScreen() { return nuvigator.pushNamed<Object>(MerchantRoutes.merchantScreen,); } Future<Object> pushReplacementToMerchantScreen<TO extends Object>({TO result}) { return nuvigator.pushReplacementNamed<Object, TO>(MerchantRoutes.merchantScreen, result: result,); } Future<Object> pushAndRemoveUntilToMerchantScreen<TO extends Object>({@required RoutePredicate predicate}) { return nuvigator.pushNamedAndRemoveUntil<Object>(MerchantRoutes.merchantScreen, predicate,); } Future<Object> popAndPushToMerchantScreen<TO extends Object>({TO result}) { return nuvigator.popAndPushNamed<Object, TO>(MerchantRoutes.merchantScreen, result: result,); } Future<Object> toMerchantPhotosScreen({List<AssetImageModel*>* photos}) { return nuvigator.pushNamed<Object>(MerchantRoutes.merchantPhotosScreen, arguments: {'photos': photos,},); } Future<Object> pushReplacementToMerchantPhotosScreen<TO extends Object>({List<AssetImageModel*>* photos, TO result}) { return nuvigator.pushReplacementNamed<Object, TO>(MerchantRoutes.merchantPhotosScreen, arguments: {'photos': photos,}, result: result,); } Future<Object> pushAndRemoveUntilToMerchantPhotosScreen<TO extends Object>({List<AssetImageModel*>* photos, @required RoutePredicate predicate}) { return nuvigator.pushNamedAndRemoveUntil<Object>(MerchantRoutes.merchantPhotosScreen, predicate, arguments: {'photos': photos,},); } Future<Object> popAndPushToMerchantPhotosScreen<TO extends Object>({List<AssetImageModel*>* photos, TO result}) { return nuvigator.popAndPushNamed<Object, TO>(MerchantRoutes.merchantPhotosScreen, arguments: {'photos': photos,}, result: result,); } 

```
@NuRoute()
ScreenRoute merchantPhotosScreen({
  List<AssetImageModel> photos,
}) => ScreenRoute(
  builder: (BuildContext context) => MerchantPhotosScreen(
    photos: photos,
  ),
);
```

**Comment by stwonary** [comment]
it's a related issue mentioned in #48.  the workaround is to downgrade to 0.7.0

**Comment by tnitsiri** [comment]
I tried but still error
build_runner: ^1.10.3
nuvigator: ^0.7.0

**Comment by leoiacovini** [comment]
We are investigating, but the problem is related to this bump:
https://github.com/nubank/nuvigator/pull/52/files#diff-8b7e9df87668ffa6a04b32e1769a33434999e54ae081c52e5d943c541d4c0d25L16-R16

Can you pin the version of the analyzer to be lower than 0.41.0 (be sure to clean the dependencies to avoid cache), and try again?

**Comment by tnitsiri** [comment]
Problem solved for my case on
analyzer: ^0.39.17
build_runner: ^1.10.2
nuvigator: ^0.7.1

**Comment by leoiacovini** [comment]
This should have been solved in the new `0.7.2` - sorry for the trouble

**Comment by tnitsiri** [comment]
Thank you, it's works fine now for all latest version.
